### DISCUSSION TITLE
Is it possible to disable the ability to genrate/use ssh keys for git auth

### DISCUSSION THREAD

**Discussion by PuneeshMotwani007** [community/community#139170] [New to GitHub]
I would like my org to use only the oAuth tokens to auth git to interact with github. Is it possible to disable ssh key auth in github?

**Comment by sam-math** [comment]
Hey Puneesh, have you checked this on the github docs:
[https://docs.github.com/en/enterprise-server@3.10/admin/managing-accounts-and-repositories/managing-repositories-in-your-enterprise/disabling-git-ssh-access-on-your-enterprise](https://docs.github.com/en/enterprise-server@3.10/admin/managing-accounts-and-repositories/managing-repositories-in-your-enterprise/disabling-git-ssh-access-on-your-enterprise)

**Reply by PuneeshMotwani007** [reply]
Hi @sam-math 

Thanks for your reply. This page say that this capability is present for Github enterprise server. Does it mean the same capability is available for github enterprise plan on cloud?

is github enterprise server same tings as the github enterprise plan on cloud?
